container_pipeline/utils.py

```diff
@@ -1,10 +1,14 @@
+import hashlib
 import os
 
 
 def get_job_name(job_details):
     """Get jenkins job name from job_detials"""
-    return str(job_details['appid']) + "-" + str(job_details['jobid']) + "-" \
-        + str(job_details['desired_tag'])
+    namespace = str(job_details['appid']) + "-" + str(job_details['jobid']) + \
+        "-" + str(job_details['desired_tag'])
+    project = hashlib.sha224(namespace).hexdigest()
+
+    return project, namespace
```

Comment:
IMO, the utility functions should be small, doing one thing at a time. If there are two purposes, lets write two functions, in actual code, we can chain the utility functions to yield required results.